src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -0,0 +1,30 @@
+<template >
+    <template if:false={isActiveInstance}>
+        <div class="container slds-is-relative">
+            <div class="spinner" if:true={isLoading}>
+                <lightning-spinner alternative-text="Loading"></lightning-spinner>
+            </div>
+            <div class="image-container"><img class="image" src={img}/></div>
+            <div if:false={isApplicationSubmitted} class="info pending">
+                <p>{labels.gsNoApplicationSubmitted} | <a>{labels.gsLearnMore}</a></p>
+                <p class="daysLeft">{diffInDays} {labels.gsDaysRemainingInFreeTrial}</p>
+                <a class="slds-button slds-button_brand" href="https://powerofus.force.com/PowerOfUsApply" target="_blank">
```

Comment:
Why not just using <lightning-button> as well as all standard SLDS base components in all of this?